bin/setup-bench.mjs

```diff
@@ -0,0 +1,155 @@
+import 'zx/globals';
+import os from 'node:os';
+import { join } from 'node:path';
+
+/*
+
+  To run proper bench setup we need to do following things:
+
+  1.) Compile control packages
+  2.) Compile experiment packages
+  3.) Use SAME benchmark source
+      * we should be able to tweak bench
+        (add more cases, and still be able to compare with control)
+      * we should be able to re-run bench in CI from current branch with updated perf source
+
+*/
+
+const experimentBranchName =
+  process.env['EXPERIMENT_BRANCH_NAME'] || (await $`git rev-parse --abbrev-ref HEAD`).stdout.trim();
+const controlBranchName = process.env['CONTROL_BRANCH_NAME'] || 'main';
+const markers =
+  process.env['MARKERS'] ||
+  'navigationStart,renderStart,renderEnd,glimmer-render-1000-rows-start,glimmer-render-1000-rows-finished';
+const fidelity = process.env['FIDELITY'] || '20';
+const throttleRate = process.env['THROTTLE'] || '4';
+const FORK_NAME = process.env['FORK_NAME'] || '';
+
+const tempDir = os.tmpdir();
+
+const CONTROL_DIR = join(tempDir, 'control');
+const EXPERIMENT_DIR = join(tempDir, 'experiment');
+
+const CONTROL_BENCH_DIR = join(CONTROL_DIR, 'benchmark', 'benchmarks', 'krausest');
+const EXPERIMENT_BENCH_DIR = join(EXPERIMENT_DIR, 'benchmark', 'benchmarks', 'krausest');
+
+const pwdRaw = await $`pwd`;
+const pwd = pwdRaw.toString().trim();
+
+// we use benchmark from current commit, very useful if we need to tweak it
+const benchmarkFolder = 'benchmark';
+
+// remove node_modules from benchmark folder, maybe we could figure out better option to distribute bench source
+await $`rm -rf ${join(pwd, benchmarkFolder, 'node_modules')}`;
```

Comment:
non-blocking, I think execa has a `$` now, idk if you need zx

Reply:
@NullVoxPopuli I was spend 4 h dealing with `cwd` issues with `zx` (in CI). At the moment I don't have mental health to backport it to execa :) , but not against if smb could unify it.

Reply:
after I finished reading this file, I'm 100% ok with zx -- the utility functions it provides look good